src/feedback.css.ts

```diff
@@ -46,6 +46,14 @@ export const innerContainer = style({
     },
 });
 
+export const feedbackScreenInverse = style({
```

Comment:
this fixes this webapp case, where the parent component was setting a white background:
![global-bundle-purchase-screenshot-test-tsx-test-bundle-purchase-3-diff](https://github.com/Telefonica/mistica-web/assets/1849576/fd914e91-d65f-4e62-81d7-a1da7a841041)